cluster/node-pools/master-default/userdata.yaml

```diff
@@ -888,5 +891,6 @@ write_files:
       kind: Configuration
       limits:
         - type: Server
-          qps: 500
-          burst: 1000
\ No newline at end of file
+          qps: {{ .Cluster.ConfigItems.event_rate_limit_config_qps }}
+          burst: {{ .Cluster.ConfigItems.event_rate_limit_config_burst }}
+{{ end }}
```

Comment:
We should add the newline here to be consistent with the other files.

Reply:
Done, thanks! 🙏